integrate/artifacts/tutorial.md

```diff
@@ -149,4 +152,88 @@ But this time we will download artifacts produced by the configuration from `Exa
             }
         }
     }
+   ```
+
+### 4. Shared artifacts
+This example will show how to share artifacts within an orchestration
+We will create two configurations of Python Transformation component.
+One will produce a shared artifact and the other will consume it. 
+Both configurations needs to be in the same orchestration.
+The configuration producing artifact needs to be in a phase that precedes the consuming one.
+
+1. Create "Producer" configuration
+   The configuration will look like this:
+
+   ```json
+   {
+       "parameters": {
+           "blocks": [
+               {
+                   "name": "Block 1",
+                   "codes": [
+                       {
+                           "name": "Upload shared",
+                           "script": [
+                               "import os\npath = \"/data/artifacts/out/shared\"\nwith open(path+\"/myartifact3\", \"w\") as file:\n  file.write(\"value1\")"
```

Comment:
Myslim ze tady by bylo uzitecnejsi nedavat dekodovanou konfiguraci, ale spis dekodovany ten Python kod (na konfiguraci bych se vyprd). Stejne jak to mas u predchozich examplu. Kdyz budu pracovat s artefaktama, potrebuju vedet, jak ma vypadat ten kod, ne struktura configu.

Reply:
done